### DISCUSSION TITLE
Différencier l’évaluation selon la taille d’écran

### DISCUSSION THREAD

**Discussion by julienrf** [cnumr/EcoIndex#298] [Ideas]
Selon la taille de l’écran du terminal client (ordinateur vs téléphone), un même site web peut afficher un contenu différent (images, mais aussi DOM), et donc avoir un impact environnemental différent.

Pensez-vous qu’il serait utile / pertinent de faire deux évaluations, une pour un petit écran, et une pour un écran de laptop ? C’est ce qui est fait dans https://pagespeed.web.dev/, par exemple.

**Comment by vvatelot** [comment]
Bonjour @julienrf , effectivement ça pourrait être intéressant. L'API permet de lancer des analyses sur différentes tailles d'écrans, donc techniquement c'est possible. Maintenant, il y a un gros travail de réflexion à mener pour faire évoluer le site:
- Techniquement, mettre à jour le service API du site internet pour stocker les infos des 2 analyses
- Comment faire pour afficher la page résultat? Actuellement elle est du type `/resultat?id=xxxxxxxxx` avec uniquement l'ID de l'analyse desktop...
- Comment faire évoluer le design de la page de résultats pour afficher les 2 résultats?

À mon avis, il ya encore d'autres questions auquelles je ne pense pas... :thinking:
